### ISSUE TITLE
JetAppointments Zoom Integration

### ISSUE THREAD

**Issue by StaffordSquier** [CrocoBlock/suggestions#7164]
Currently, when you add Zoom integration to jetAppointments, it applies to all services and service providers. It would be great if this functionality could be applied at the individual service and provider level. 

I run a company that delivers both online and in-person training. I need to be able to assign the Zoom functionality to only those services that will be delivered online. Currently, the Zoom integration is applied to every service and service provider.

**Comment by Marcoqwerty78** [comment]
I agree! i have the same request. Appointment must be dove Virtual or Live

**Comment by SamiJ3091** [comment]
+1
